generate-err-logs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+output_file="deployment-err.log"
+> $output_file
+
+namespaces=$(kubectl get namespaces -o=jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}')
```

Comment:
`kubectl get namespace -o name` will be simpler

Reply:
changed, but it required getting rid of the `namespace/` prefix so I used `xargs -n1 basename`